templates/lookup/ifupdown__dynamic_packages.j2

```diff
@@ -16,6 +16,4 @@
 {%     endif %}
 {%   endfor %}
 {% endif %}
-{% if ifupdown__tpl_packages %}
 {{ ifupdown__tpl_packages | unique | to_yaml }}
-{% endif %}
```

Comment:
Ah, sorry for the misunderstanding. My bad. Yes, I did test that with 2.1.3 and 2.2. I did not work for you? Can you recheck?